### DISCUSSION TITLE
Error: Please install `npm` before continuing

### DISCUSSION THREAD

**Discussion by TajangSec** [vercel/community#5555] [Help] [answered]
### Question
I met the problem when I create a new project and have no idea about how to fix it.

- vercel project deployment: https://vercel.com/tajangsec/waline-test3/AhH9JFzDNjNDjwvmgw8dXDqi8V8g
- GitHub repo: https://github.com/TajangSec/waline-test3

```
Running build in Washington, D.C., USA (East) – iad1
22:32:49.476 | Cloning github.com/TajangSec/waline-test3 (Branch: main, Commit: 8fff613)
22:32:49.737 | Cloning completed: 260.774ms
22:32:49.824 | Previous build cache not available
22:32:50.011 | Running "vercel build"
22:32:50.510 | Vercel CLI 33.2.0
22:32:50.659 | WARN! Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
22:32:50.667 | > Installing Builder: @vercel/node@2.5.10
22:32:50.678 | Error: Please install `npm` before continuing
22:32:50.678 | Learn More: https://vercel.link/builder-dependencies-install-failed
22:32:50.861
```
![Clip_2024-01-22_15-45-02](https://github.com/vercel/community/assets/63721558/99dc6c4e-fc57-49f7-8b36-e60eeb0fe6e4)

### Example

https://github.com/TajangSec/waline-test3

### Steps to Reproduce

The official waline tutorial：https://waline.js.org/en/guide/get-started/#deploy-to-vercel-server
Click the deploy button to deploy：
![Clip_2024-01-22_15-38-51](https://github.com/vercel/community/assets/63721558/5c067c1f-6d91-4d32-a5f6-cbab132c6c19)
I can't deploy my account, the link to the repository is: https://github.com/TajangSec/waline-test3 I can provide the vercel account to reproduce it if needed!

**Comment by swarnava** [comment]
You no longer need to have `@vercel/node` in your vercel.json. Can you try without it? For specific template, you can try reaching out to their support team.

**Reply by TajangSec** [reply]
Thanks for the reply, but it didn't work

**Reply by swarnava** [reply]
I just deployed their example [template](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample) mentioned in their [docs](https://waline.js.org/en/guide/deploy/vercel.html#how-to-deploy) and it seems deployed successfully: https://walinejs.preview.vercel-support.app/

Are you making any specific changes?

**Reply by TajangSec** [reply]
No changes were made and the tutorial was followed exactly step by step. But I'm having problems, and I think it's because my github or vercel is set up wrong in some places.
I will have the developer team logged into my Vercel platform, and when they deployed they reproduced my Error, they were amazed, and they don't know exactly why.
Their developers submitted another work order to Vercel: https://github.com/vercel/vercel/discussions/11072
But both work orders were ignored by Vercel.

**Reply by lizheming** [reply]
@swarnava Hello, I am the developer of this project. It seems to be related to the account. I have no problem deploying using my own account, but there is a problem deploying using @TajangSec's account. And many users have reported similar problems.

**Reply by swarnava** [reply, marked as answer]
We have tweaked a change, Can you try again and let us know if you are still facing the issue?

**Reply by TajangSec** [reply]
It worked. Thank you so much!
![Clip_2024-01-25_18-33-21](https://github.com/vercel/community/assets/63721558/c4565516-18a8-4e28-96fd-14cb3cdfb632)
